test/app-tests/loading.js

```diff
@@ -47,7 +47,7 @@ describe('loading:', function () {
     // the mounted MatrixChat
     let matrixChat;
 
-    // a promise which resolves when the MatrixChat calls onLoadCompleted
+    // a promise which resolves when the MatrixChat calls onTokenLoginCompleted
```

Comment:
Ah only the token login test uses this promise. Could you rename the promise in that case? :innocent: